src/Tools/_framework/Footers/MathInputKeyboard.jsx

```diff
@@ -231,141 +246,564 @@ export default function VirtualKeyboard() {
   // if (toggleLetters) {
   // if (toggleCase) {
   let sectionUpperABC = (
-    <Panel tabIndex="0" ref={containerRef}>
-      <LettersSection>
-        <LetterButton onClick={() => callback("write Q")}>Q</LetterButton>
-        <LetterButton onClick={() => callback("write W")}>W</LetterButton>
-        <LetterButton onClick={() => callback("write E")}>E</LetterButton>
-        <LetterButton onClick={() => callback("write R")}>R</LetterButton>
-        <LetterButton onClick={() => callback("write T")}>T</LetterButton>
-        <LetterButton onClick={() => callback("write Y")}>Y</LetterButton>
-        <LetterButton onClick={() => callback("write U")}>U</LetterButton>
-        <LetterButton onClick={() => callback("write I")}>I</LetterButton>
-        <LetterButton onClick={() => callback("write O")}>O</LetterButton>
-        <LetterButton onClick={() => callback("write P")}>P</LetterButton>
-        <LetterButton onClick={() => callback("write A")}>A</LetterButton>
-        <LetterButton onClick={() => callback("write S")}>S</LetterButton>
-        <LetterButton onClick={() => callback("write D")}>D</LetterButton>
-        <LetterButton onClick={() => callback("write F")}>F</LetterButton>
-        <LetterButton onClick={() => callback("write G")}>G</LetterButton>
-        <LetterButton onClick={() => callback("write H")}>H</LetterButton>
-        <LetterButton onClick={() => callback("write J")}>J</LetterButton>
-        <LetterButton onClick={() => callback("write K")}>K</LetterButton>
-        <LetterButton onClick={() => callback("write L")}>L</LetterButton>
-        <White15Button onClick={handleToggleABCCase}>
+    <Box
+      ref={containerRef}
+      // height="240px"
+      bottom="0"
+      left="0"
+      width="100%"
+      /* background-color: var(--canvas); */
+      /* color: var(--canvas); */
+      display="flex"
+      flexDirection="row"
+      textAlign="center"
+      justifyContent="center"
+    >
+      <Box
+        height="150px"
+        maxWidth="600px"
+        flexBasis="90%"
+        marginLeft="5px"
+        marginRight="5px"
+        marginTop="auto"
+        marginBottom="auto"
+        display="flex"
+        flexWrap="wrap"
+        justifyContent="space-evenly"
+      >
+        <Button
```

Comment:
Instead of making all of these changes, do you want to just create a small function component called LetterButton that applies flexBasis and variant and replaces the previous styled component? Then you can avoid most of these changes and these would stay as a single line. And if we ever need to apply something to all of these type of buttons we would have one place to do it again.